src/gui/app.cpp

```diff
@@ -159,6 +163,10 @@ void App::processCommandlineOptions(QCommandLineParser& parser, QStringList argu
 				QCoreApplication::translate("main", "Error if nvim does not responde after count milliseconds"),
 				QCoreApplication::translate("main", "ms"),
 				"20000"));
+	parser.addOption(QCommandLineOption("class",
```

Comment:
I wouldn't use `WM_CLASS` in the description; this is only correct for Linux + X11. Also capitalization.

Perhaps something more generic: `Set a custom application name`?